spec/support/data_api/shared_examples.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.shared_examples 'a TAD API endpoint' do |path|
+  it 'verifies the API token' do
```

Comment:
I think the logic here is now deprecated: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/spec/requests/data_api/tad_api_spec.rb as some of it is covered by the above specs. Maybe we can move the other authorisation specs for other API users etc. into here and remove the old spec

Reply:
Good shout. I've removed the `tad_api_spec.rb` file. Let's look at refactoring post-Register?